code/app/src/main/java/com/example/snackoverflow/MainActivity.java

```diff
@@ -31,13 +32,22 @@ protected void onCreate(Bundle savedInstanceState) {
         String []categories = {"Lunch","Dinner"};
 
 
-        //Test Data
+        //Recipe test
         for (int i =0;i<recipestitle.length;i++){
             recipeDataList.add(new Recipe(recipestitle[i], LocalTime.now(),2.0f,"Lunch","HAHA",new ArrayList<String>(Arrays.asList(new String[]{"Nidal","Nasemm"}))));
         }
-
         recipeArrayAdapter = new RecipeAdapter(this,recipeDataList);
         recipeList.setAdapter(recipeArrayAdapter);
+        Button recipeadd = findViewById(R.id.storage_activity2);
```

Comment:
Look at my other comment about a duplicate

Reply:
i made changes to her files to make a test for recipe